wordpress-mobile/WordPress-iOS
# Theme activation and option details

Closes #3906; closes #3907; closes #3908

Completes theme user experience to match Android implementation

To test: 
1. Enable themes feature with wpdebug://themes?enabled=1 as discussed in #3913
2. Activate themes browser from the blog details screen
3. Check that detail views available for active theme and inactive themes match Android implementation
4. Currently themes may not be purchased through the native apps: check that free and previously purchased themes are available in the browser, and unpurchased themes are not
5. Activate an inactive theme and check that activation is reflected in the browser and on the site
6. Review in general for acceptable congruence with Android themes functionality

@kwonye Please compare to your Android work!
Needs Review: @kwonye
